Minutes of meeting 4 (#17)
marda-alliance/metadata_extractors: meetings/4-metting-2023-may-05-02/minutes.md

# Progress meeting 4

02/05/2023

## Agenda

Intro from Peter
ELN roundtable discussion of feature wishlist

## Minutes

Peter Kraus: Introduced the current state of the registry, API prototype and schema.

Suggestions from Kevin Jablonka:

- Reusability and validation as main focus: is the parser tested, how is it
  tested and how reliable is it?
- Able to use code without providing much boilerplate

> Possibility of MaRDA permanent infrastructure for running the registry
> 
> Output schema is still a target but perhaps a distant one

Reviewer: Perhaps make clear this is not coming from Kevin but from Me:

```suggestion
Peter:
- possibility of MaRDA permanent infrastructure for running the registry
- output schema is still a target but perhaps not in current WG
```
